packages/rrweb/src/record/index.ts

```diff
@@ -291,13 +309,15 @@ function record<T = eventWithTime>(
     adoptedStyleSheetCb: wrappedAdoptedStyleSheetEmit,
   });
 
-  const iframeManager = new IframeManager({
-    mirror,
-    mutationCb: wrappedMutationEmit,
-    stylesheetManager: stylesheetManager,
-    recordCrossOriginIframes,
-    wrappedEmit,
-  });
+  const iframeManager: IframeManagerInterface = __RRWEB_EXCLUDE_IFRAME__
```

Comment:
We have to do these checks with `typeof __RRWEB_EXCLUDE_IFRAME__ !== 'undefined'` otherwise this will throw during runtime because the binding is not defined.

Reply:
hmm, I was originally planning to do that in the sentry SDK build, but maybe it is safer to do it here 🤔 👍

Reply:
updated it to check for this!